src/drivers/driver_zephyr.c

```diff
@@ -1580,8 +1580,8 @@ int wpa_drv_zep_stop_ap(void *priv)
 	}
 
 	if_ctx->freq = 0;
-	if_ctx->beacon_set = false;
 out:
+	if_ctx->beacon_set = false;
```

Comment:
Should this be protected with `if (if_ctx) {` (since this could crash for the `if (!priv) {` conditional code